### PR TITLE
fix: preserve Windows newline characters

### PR DESCRIPTION
**Pull request by philippfromme**
Related to camunda/camunda-modeler#2280

### CODE REVIEW

File: lib/TextBox.js [hunk 1]
```diff
@@ -452,3 +454,13 @@ TextBox.prototype.setSelection = function(container, offset) {
   selection.removeAllRanges();
   selection.addRange(range);
 };
+
+// helpers //////////
+
+function normalizeEndOfLineSequences(string, endOfLineSequence) {
+  if (isUndefined(endOfLineSequence)) {
```
Comment (nikku) [inline lib/TextBox.js:461]: How can `endOfLineSeuqence` ever be undefined here?

Comment (philippfromme) [reply lib/TextBox.js:461]: What this does is effectively default to LF. I can remove that switch if it's considered unnecessary.

Comment (nikku) [reply lib/TextBox.js:461]: How does that second parameter sometimes exist? Is it `'FOO'.replace(/OO/, function(o, o) { })` ?

Comment (philippfromme) [reply lib/TextBox.js:461]: The way we use it (`text = normalizeEndOfLineSequences(text);`) it never exists, therefore it defaults to LF.

Comment (nikku) [reply lib/TextBox.js:461]: That one is super confusing to be honest. Let's always default to `\n` if that does not break anything. The second parameter is completely unused, too. I'd suggest to get rid of it.

Comment (philippfromme) [reply lib/TextBox.js:461]: Ok, will do! 👍🏻

Comment (philippfromme) [reply lib/TextBox.js]: Done. 👍🏻

**Review by nikku — APPROVED** [review]

**Pull request merged**